### ISSUE TITLE
Support for Additional Mods (1.16.5)

### ISSUE THREAD

**Issue by supermassimo** [supermassimo/DynamicTrees-ExC#59]
With Dynamic Trees for 1.16 being released, the door to 1.16 mod compatibility opens.
Following the same rules as the [1.12 suggestion list](https://github.com/supermassimo/DynamicTrees-ExC/issues/41), comment under this post 1.16 mods that could have Dynamic Trees compatibility.
Suggestions from the [discord server](https://discord.gg/83qcpkY) will also be added here.

**Please please please include how many trees a mod adds when suggesting**

> DO NOT suggest the following types of mods as they will not be considered:
>  - Mods without a public source (such as MCreator mods)

Those in **bold** were considered and/or are being worked on.
When a mod is finished and published, it is removed from this list.

The stars are a rough estimate of how likely it is for me (subjectively, anyone is free to pick any!) to choose that mod.
It is based on how popular the mod is, how many trees it has, how unique these are and how interesting I find them.
**0 stars just means i do not know enough about the mod, not that i do not like it**.

**Update from 1.12:** (add-ons from 1.12 that have yet to be updated)
- **★★★★★[Biomes o' Plenty](https://www.curseforge.com/minecraft/mc-mods/biomes-o-plenty) (many trees)**
- **★★★★★[Integrated Dynamics](https://www.curseforge.com/minecraft/mc-mods/integrated-dynamics) (1 tree)**
- ★★★★★[Mystical World](https://www.curseforge.com/minecraft/mc-mods/mystical-world) (1 tree)
- **★★★★☆[Nature's Aura](https://www.curseforge.com/minecraft/mc-mods/natures-aura) (1 tree)**
- ★★★★☆[Tech Reborn](https://www.curseforge.com/minecraft/mc-mods/techreborn) (1 tree)
- ★★★☆☆[Traverse](https://www.curseforge.com/minecraft/mc-mods/traverse-reforged) (many trees)
- **★★☆☆☆[Vampirism](https://www.curseforge.com/minecraft/mc-mods/vampirism-become-a-vampire) (2 trees)**
- ★★★★★[Pam’s HarvestCraft 2](https://www.curseforge.com/minecraft/mc-mods/pams-harvestcraft-2-trees) (49 trees)
- ★★★★★[The Twilight Forest](https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest) (8 trees)
- ★★★★★[The Aether](https://www.curseforge.com/minecraft/mc-mods/the-aether) (3 trees)
- **★★★★★[Tinker's Construct](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct) (3 trees)**
- **★★★★★[Atum 2](https://www.curseforge.com/minecraft/mc-mods/atum) (3 trees)**

**New addons:**
- ★★★★☆[Pokecube AIO](https://www.curseforge.com/minecraft/mc-mods/pokecube-aoi) (17 trees)
- ★★★☆☆[Druidcraft](https://www.curseforge.com/minecraft/mc-mods/druidcraft) (2 trees)
- **★★★★☆[Oh the Biomes You'll Go](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go) (+40 trees)**
- ☆☆☆☆☆[Autumnity](https://www.curseforge.com/minecraft/mc-mods/autumnity) (4 trees)
- ☆☆☆☆☆[Ars Nouveau](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau) (4 trees)
- ☆☆☆☆☆[Atmospheric](https://www.curseforge.com/minecraft/mc-mods/atmospheric) (6 trees)
- ☆☆☆☆☆[Architect's Palette](https://www.curseforge.com/minecraft/mc-mods/architects-palette) (1 tree)
- ☆☆☆☆☆[Better End Forge](https://www.curseforge.com/minecraft/mc-mods/betterend-forge-port) (7 trees)
- ☆☆☆☆☆[Croptopia](https://www.curseforge.com/minecraft/mc-mods/croptopia-fabric) (26 trees)
- ☆☆☆☆☆[Simply Tea](https://www.curseforge.com/minecraft/mc-mods/simply-tea) (1 tree)
- ☆☆☆☆☆[Upgrade Aquatic](https://www.curseforge.com/minecraft/mc-mods/upgrade-aquatic) (1 tree)
- ☆☆☆☆☆[TerraForged](https://www.curseforge.com/minecraft/mc-mods/terraforged) (biomes)
- ☆☆☆☆☆[Enhanced Mushrooms](https://www.curseforge.com/minecraft/mc-mods/enhanced-mushrooms) (2 mushrooms)
- ☆☆☆☆☆[Greek Fantasy](https://www.curseforge.com/minecraft/mc-mods/greek-fantasy)  (1 tree)
- ☆☆☆☆☆[Silent Gear](https://www.curseforge.com/minecraft/mc-mods/silent-gear) (1 tree)
- ☆☆☆☆☆[Rediscovered](https://www.curseforge.com/minecraft/mc-mods/rediscovered) (1 tree)
- ☆☆☆☆☆[Premium Wood](https://www.curseforge.com/minecraft/mc-mods/premium-wood) (7 trees)
- ☆☆☆☆☆[Simple Farming](https://www.curseforge.com/minecraft/mc-mods/simple-farming) (9 trees)
- ☆☆☆☆☆[Tofucraft](https://www.curseforge.com/minecraft/mc-mods/tofucraftreload) (2 trees)
- ☆☆☆☆☆[Terra Incognita](https://www.curseforge.com/minecraft/mc-mods/terraincognita) (1 tree)
- ☆☆☆☆☆[Neapolitan](https://www.curseforge.com/minecraft/mc-mods/neapolitan) (1 tree)
- ☆☆☆☆☆[Explorercraft](https://www.curseforge.com/minecraft/mc-mods/explorercraft-worldexpansion) (3 trees)
- ☆☆☆☆☆[Environmental](https://www.curseforge.com/minecraft/mc-mods/environmental) (6 trees)
- ☆☆☆☆☆[Omni](https://www.curseforge.com/minecraft/mc-mods/omni) (1 tree 1 cactus)
- ☆☆☆☆☆[The Endergetic Expansion](https://www.curseforge.com/minecraft/mc-mods/endergetic) (1 tree)
- ☆☆☆☆☆[Desolation](https://www.curseforge.com/minecraft/mc-mods/desolation-forge) (1 tree)
- ★☆☆☆☆[Ice and Fire](https://www.curseforge.com/minecraft/mc-mods/ice-and-fire-dragons) (3 branches)
- ☆☆☆☆☆[Ex Nihilo: Sequentia](https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-sequentia) (1 tree (not yet implemented))
- ☆☆☆☆☆[Fruitful](https://www.curseforge.com/minecraft/mc-mods/fruitful) (1 tree)
- ☆☆☆☆☆[Bayou Blues](https://www.curseforge.com/minecraft/mc-mods/bayou-blues) (1 tree)
- ☆☆☆☆☆[Abundance](https://www.curseforge.com/minecraft/mc-mods/abundance) (2 trees, 1 cactus)
- ☆☆☆☆☆[Better Badlands](https://www.curseforge.com/minecraft/mc-mods/better-badlands) (1 tree)
- ☆☆☆☆☆[The Outer End](https://www.curseforge.com/minecraft/mc-mods/the-outer-end) (1 tree)
- ☆☆☆☆☆[Outvoted](https://www.curseforge.com/minecraft/mc-mods/outvoted) (2 trees)
- ☆☆☆☆☆[Alex's Mobs](https://www.curseforge.com/minecraft/mc-mods/alexs-mobs) (1 tree for missing banana drops)
- ☆☆☆☆☆[Ultra Amplified Dimension](https://www.curseforge.com/minecraft/mc-mods/ultra-amplified-mod) (1 cactus, dimensional world gen)
- ☆☆☆☆☆[Fruit Trees](https://www.curseforge.com/minecraft/mc-mods/fruit-trees) (9 trees)
- ☆☆☆☆☆[Terra World Generator](https://www.curseforge.com/minecraft/mc-mods/terra-world-generator) (world gen, possibly new species)
- ☆☆☆☆☆[Infernal Expansion](https://www.curseforge.com/minecraft/mc-mods/infernal-expansion) (1 fungus)
- ☆☆☆☆☆[Caves & Cliffs backport](https://www.curseforge.com/minecraft/mc-mods/caves-and-cliffs-backport) (1 tree)
- ☆☆☆☆☆[Darker Depths](https://www.curseforge.com/minecraft/mc-mods/darker-depths) (1 tree)
- ☆☆☆☆☆[Maiden's Marvelous Materials](https://www.curseforge.com/minecraft/mc-mods/maidens-marvelous-materials) (8 trees)
- ☆☆☆☆☆[The Undergarden](https://www.curseforge.com/minecraft/mc-mods/the-undergarden) (2 trees, 1 fungus, 4 mushroom)
- ☆☆☆☆☆[Malum](https://www.curseforge.com/minecraft/mc-mods/malum) (1 tree)
- ☆☆☆☆☆[Apple Trees Revived](https://www.curseforge.com/minecraft/mc-mods/apple-trees-revived) (2 trees)

**Comment by MagmaSlime123** [comment]
Update the BoP addon. You already made it so you know how many trees it adds. BoP did end up removing some biomes I think but you pretty much did the tree count for that one for us.

**Comment by supermassimo** [comment]
> Update the BoP addon. You already made it so you know how many trees it adds. BoP did end up removing some biomes I think but you pretty much did the tree count for that one for us.

I had nothing to do with the BoP addon development, please follow the post rules or you will be prevenented from participating

**Comment by WenXin20** [comment]
[Pokecube AIO](https://www.curseforge.com/minecraft/mc-mods/pokecube-aoi) adds 12 berry trees, which start out as crops when planted on farmland and then become trees after reaching the 7th stage of growth. The leaves then produce berries underneath it.  It also adds 4 other saplings, the inverted saplings (upside down once grown), temporal saplings, aged saplings, and distortic saplings. There is also a legendary tree that spawns, but that might be too big to be dynamic.
So 16 trees in total.

**Comment by KawaiiPC** [comment]
[oh the biomes you'll go](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go) It is a biome mod that adds more than 80 biomes and adds more than 40 unique trees.

**Comment by SureinDragon** [comment]
[Autumnity](https://www.curseforge.com/minecraft/mc-mods/autumnity) adds in 4 trees, but they're all pretty much the same aside leaf color.
[Ars Nouveau](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau) adds in 4 magical trees.

**Comment by WenXin20** [comment]
[Atmospheric](https://www.curseforge.com/minecraft/mc-mods/atmospheric) - 6 saplings
[Architect's Palette](https://www.curseforge.com/minecraft/mc-mods/architects-palette) - Twisted Sapling
[Better End Forge](https://www.curseforge.com/minecraft/mc-mods/betterend-forge-port) - At least 7 different trees/saplings for the End.
[Croptopia](https://www.curseforge.com/minecraft/mc-mods/croptopia-fabric) - 26 different fruit and nut Saplings.
[Simply Tea](https://www.curseforge.com/minecraft/mc-mods/simply-tea) - 1 Sapling
[Upgrade Aquatic](https://www.curseforge.com/minecraft/mc-mods/upgrade-aquatic) - River Sapling and Driftwood (logs only)

**Comment by RealAlax** [comment]
[TerraForged](https://www.curseforge.com/minecraft/mc-mods/terraforged) is a world generation mod for Minecraft that creates more immersive, inspiring worlds
to explore. It overhauls the vanilla generation system with all new terrain, better rivers,
custom biome features and tonnes of configuration options to boot!

It adds no new trees, but it contains modified vanilla trees.

**Comment by Dalethmann** [comment]
[Pam's HarvestCraft 2 - Trees](https://github.com/MatrexsVigil/phc2trees)
Total Trees Added (49) (3 variants) Pretty much all the trees use the same model depending on temperate, warm and cold climates.

Temperate (Fruit) - Apple, Avocado, Candlenut, Cherry, Chestnut, Gooseberry, Lemon, Nutmeg, Orange, Peach, Pear, Plum, Walnut, Spiderweb, Hazelnut, Pawpaw, Soursop

Warm (Fruit) -Almond, Apricot, Banana, Cashew, Coconut, Date, Dragonfruit, Durian, Fig, Grapefruit, Lime, Mango, Olive, Papaya, Pecan, Peppercorn, Persimmon, Pistachio, Pomegranate, Starfruit, Vanilla Bean, Breadfruit, Guava, Jackfruit, Lychee, Passionfruit, Rambutan, Tamarind

Warm (Log Fruit) - Cinnamon, Paperbark

Cold (Fruit) - Pinenut

Cold (Log Fruit) - Maple

**Comment by AtaqanKurt** [comment]
Enhanced mushrooms makes mushrooms a tree type (idk if that counts)

**Comment by supermassimo** [comment]
> Enhanced mushrooms makes mushrooms a tree type (idk if that counts)

No it doesn’t, This post is for other mods. Vanilla additions are in another post.

**Comment by WenXin20** [comment]
[Greek Fantasy](https://www.curseforge.com/minecraft/mc-mods/greek-fantasy) - Olive Sapling
[Silent Gear](https://www.curseforge.com/minecraft/mc-mods/silent-gear) - Netherwood Sapling
[Rediscovered](https://www.curseforge.com/minecraft/mc-mods/rediscovered) - Cherry Sapling
[Premium Wood](https://www.curseforge.com/minecraft/mc-mods/premium-wood) - 7 Saplings
[Simple Farming](https://www.curseforge.com/minecraft/mc-mods/simple-farming) - 9 Fruit Saplings
[Tofucraft](https://www.curseforge.com/minecraft/mc-mods/tofucraftreload) - 2 Tofu Saplings

**Comment by WenXin20** [comment]
[Terra Incognita](https://www.curseforge.com/minecraft/mc-mods/terraincognita) has an apple tree. (Plus a new Hazel Nut tree)

**Comment by WenXin20** [comment]
[Neapolitan](https://www.curseforge.com/minecraft/mc-mods/neapolitan) - Banana Trees grown from banana fronds.
[Explorercraft](https://www.curseforge.com/minecraft/mc-mods/explorercraft-worldexpansion) - 3 trees
[Environmental](https://www.curseforge.com/minecraft/mc-mods/environmental) - 6 trees
[Omni](https://www.curseforge.com/minecraft/mc-mods/omni) - 1 tree and Saguaro cacti

**Comment by Cr4fTJuNkiE** [comment]
Oh The Biomes You Go PLEASE! Every bit as good as BOP! Thank you.

**Comment by Luigipunch** [comment]
Gonna just throw all of team aurora's tree mods in here:

[Fruitful](https://www.curseforge.com/minecraft/mc-mods/fruitful) - 1 variant of oak that bears apples
[Bayou Blues](https://www.curseforge.com/minecraft/mc-mods/bayou-blues) - 1 aquatic tree that can grow mossy vines and gooseberries
[Abundance](https://www.curseforge.com/minecraft/mc-mods/abundance) - 2 trees and 1 type of cactus
[Better Badlands](https://www.curseforge.com/minecraft/mc-mods/better-badlands) - 1 smaller variant of dark oak
[Enhanced Mushrooms](https://www.curseforge.com/minecraft/mc-mods/enhanced-mushrooms) - overhauls vanilla giant mushrooms and quark glowshrooms

**Comment by Dradon7** [comment]
[The Outer End](https://www.curseforge.com/minecraft/mc-mods/the-outer-end) - 1 Azure Tree in the end

Also, I just want to second my support for the abnormals mods

**Comment by SalehGrey** [comment]
[Outvoted](https://www.curseforge.com/minecraft/mc-mods/outvoted) - Adds two new trees, Baobab and Palm trees.

**Comment by Beam196** [comment]
**alex's mobs https://www.curseforge.com/minecraft/mc-mods/alexs-mobs** add banana trees to drop bananas intead of drop them from jungle leaves

**Comment by WenXin20** [comment]
[Ultra Amplified Dimension](https://www.curseforge.com/minecraft/mc-mods/ultra-amplified-mod) - 1 new giant cactus and bigger versions of vanilla trees in the new dimension.

**Comment by OrangeBurrito** [comment]
[Fruit Trees](https://www.curseforge.com/minecraft/mc-mods/fruit-trees) - Adds nine trees that have the same vanilla oak tree layout and grow different fruits.  The leaves are similar colors with the exception of the cherry tree, which 
generates a thick mat of leaf carpet below it.

**Comment by Harleyoc1** [comment]
> [Fruit Trees](https://www.curseforge.com/minecraft/mc-mods/fruit-trees) - Adds nine trees that have the same vanilla oak tree layout and grow different fruits. The leaves are similar colors with the exception of the cherry tree, which
> generates a thick mat of leaf carpet below it.
> 
> [Forbidden and Arcanus](https://www.curseforge.com/minecraft/mc-mods/forbidden-arcanus) - Adds two trees, one is tall and spindly with pink leaves like an acacia tree (and drops cherry peaches) and the other spawns in dark forest biomes and has no leaves.
> 
> [The Twilight Forest](https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest) - Adds eight trees, some of which are very thick and tall, and others are tall one-block trees that generate fireflies on them. The rainbow tree has a special morphing rainbow color on its leaves.
> 
> [Mystical World](https://www.curseforge.com/minecraft/mc-mods/mystical-world) - Adds one charred tree that has no leaves.
> 
> [Biomes O' Plenty](https://www.curseforge.com/minecraft/mc-mods/biomes-o-plenty) - Adds seventeen different trees, some of which use regular oak/birch wood but have special leaves. There are redwood trees that have unique wood + leaves, and have 2x2 thick logs with a larger base. Generates bushes in biomes such as the scrubland. Dead trees generate with their own special wood but have no leaves. Fir trees generate as tall and spindly similar to spruce but with small leaves extruding in segments and a slight tapering at the top.
> Jacaranda, Autumn Yellow/Orange, Maple, Flowering and Magic Trees generate similar to Oak trees, but the magic tree emits bright blue particles going downwrads from its leaves.
> 
> [Tinker's Construct](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct) - Adds slime trees that generate on slime islands.

Please read the list before making suggestions (it would save both of us some time)... The last four are already listed.

**Comment by OrangeBurrito** [comment]
Ah, my apologies! I assumed the "update from 1.12" section meant the addons were for 1.12 only.  You can delete these extraneous comments.

**Comment by WenXin20** [comment]
[Infernal Expansion](https://www.curseforge.com/minecraft/mc-mods/infernal-expansion) - Luminous Fungus which grows into a tree when planted & bonemealed on shimmer sand .

**Comment by WenXin20** [comment]
[The Undergarden](https://www.curseforge.com/minecraft/mc-mods/the-undergarden) - A new dimension with 2 new trees, ~1 new fungus~ (changed to a tree), and 4 new giant mushroom types.
[Forbidden Arcanus](https://www.curseforge.com/minecraft/mc-mods/forbidden-arcanus) - 2 new trees/saplings, plus the Edelwood Log (no sapling)

**Comment by supermassimo** [comment]
> [The Undergarden](https://www.curseforge.com/minecraft/mc-mods/the-undergarden) - A new dimension with 2 new trees, 1 new fungus, and 4 new giant mushroom types.
> [Forbidden Arcanus](https://www.curseforge.com/minecraft/mc-mods/forbidden-arcanus) - 2 new trees/saplings, plus the Edelwood Log (no sapling)

Forbidden arcanus is already done, im just waiting on the last DT beta

**Comment by CyanWolfBR** [comment]
[Apple Trees Revived](https://www.curseforge.com/minecraft/mc-mods/apple-trees-revived) - Adds a simple Feature to get apples in a realistic way, exist 3 types of saplings, each one generates an apple tree of both types of apples existing in minecraft: "Apple Tree Sapling" & "Gold Apple Tree Sapling" and "Notch Apple Tree Sapling" (disable by default)

**Comment by MrDenn** [comment]
The Endergetic Expansion
https://www.curseforge.com/minecraft/mc-mods/endergetic
Adds 1 new tree in the end, included in a beautiful new biome.

**Comment by Harleyoc1** [comment]
@MrDenn this is already on the list.

**Comment by Brun333rp** [comment]
I think it would be easy to implement banana trees for Alex's Mobs, as they could just re-use the design of banana trees from DT addon for Pam's Harvest Craft. unless if a diferentiation between them is desired

**Comment by supermassimo** [comment]
> I think it would be easy to implement banana trees for Alex's Mobs, as they could just re-use the design of banana trees from DT addon for Pam's Harvest Craft. unless if a diferentiation between them is desired

Yes, its already on the list

**Comment by Brun333rp** [comment]
> Yes, its already on the list

yes, I know. I just commented about it

**Comment by zoimos** [comment]
[Better Default Biomes](https://www.curseforge.com/minecraft/mc-mods/better-default-biomes) - Two trees, Palm and Willow

**Comment by RangerArden** [comment]
[https://www.curseforge.com/minecraft/mc-mods/the-lord-of-the-rings-mod-renewed](Lord of the Rings Mod Renewed) Adds numerous tree variants in a separate dimension from the overworld, it'd be *really* cool to get those dynamic-fied. Reiterate how great it'd be to have Druidcraft too.

**Comment by Brun333rp** [comment]
> Reiterate how great it'd be to have Druidcraft too.

Druidcraft is already on the list

**Comment by rowanthemodder** [comment]
[Occultism (Link to curseforge)](https://www.curseforge.com/minecraft/mc-mods/occultism)
Two trees (Unstable and stable) and some special behavior with masking their appearance for unstable otherworld trees. Special behavior leads me to believe this will be a bit of a weird one to work on.

**Comment by MrDenn** [comment]
The "Organics" mod (https://www.curseforge.com/minecraft/mc-mods/organics)
Adds 2 new biomes with 2 corresponding trees: stone and hoaky trees

**Comment by WenXin20** [comment]
[Mystical Biomes](https://www.curseforge.com/minecraft/mc-mods/mystical-biomes) - Sprout Forest for Mystical World, but oak trees that spawn are not dynamic.
[More Minecarts and Rails](https://www.curseforge.com/minecraft/mc-mods/more-minecarts) - Glass Cactus that can only be planted in desert type biomes.

**Comment by WilliamBlaze** [comment]
[Pam's Harvestcraft 2 Trees](https://www.curseforge.com/minecraft/mc-mods/pams-harvestcraft-2-trees) -it's literally all of the old Pam's harvestcraft 2 Trees, but remade for 1.16.5 as its own mod that can be used optionally with or without the core Harvestcraft 2 Mods - because apparently Pam found it easier to split the portions up into smaller work loads'

**Comment by supermassimo** [comment]
> [Pam's Harvestcraft 2 Trees](https://www.curseforge.com/minecraft/mc-mods/pams-harvestcraft-2-trees) -it's literally all of the old Pam's harvestcraft 2 Trees, but remade for 1.16.5 as its own mod that can be used optionally with or without the core Harvestcraft 2 Mods - because apparently Pam found it easier to split the portions up into smaller work loads'

Pam’s harvestcraft 2 is already listed.

**Comment by DarthCaesium** [comment]
[The Endergetic Expansion](https://curseforge.com/minecraft/mc-mods/endergetic) - Adds 1 tree, the Poise Tree. The developers will probably add more trees in the future.

**Comment by supermassimo** [comment]
> [The Endergetic Expansion](https://curseforge.com/minecraft/mc-mods/endergetic) - Adds 1 tree, the Poise Tree. The developers will probably add more trees in the future.

The Endergetic Expansion is already on the list.

**Comment by DarthCaesium** [comment]
About [Better End Forge](https://www.curseforge.com/minecraft/mc-mods/betterend-forge-port)'s Dynamic Trees addon, which is on the list of new addons, the mod also has a cactus, namely the Neon Cactus. If you could rewrite it to say "(7 trees, 1 cactus)", I would appreciate it. 

Also, the mod [Organics](https://www.curseforge.com/minecraft/mc-mods/organics) does not have a publicly viewable source code, so there is no point in adding it to the list.

**Comment by Brun333rp** [comment]
[Better Badlands](https://www.curseforge.com/minecraft/mc-mods/better-badlands) also adds Saguaro cacti, wich were previously a feature from [Abundance](https://www.curseforge.com/minecraft/mc-mods/abundance) and have now been moved to this mod. So it should be rewritten to say "1 tree, 1 cactus", and Abundance to say just "2 trees".

**Comment by Brun333rp** [comment]
Also, [Terra Ingonita](https://www.curseforge.com/minecraft/mc-mods/terraincognita) adds 2 trees now, with both being Apple and Hazel trees. So it should be updated to say "2 trees"

**Comment by lebagvondouche** [comment]
Hi, would love to see support for [Pollinate!](https://www.curseforge.com/minecraft/mc-mods/pollinate)!

**Comment by Brun333rp** [comment]
> Hi, would love to see support for Pollinate!!

seems like this is a MCreator. it's stated that these type of mods won't receive support

**Comment by lebagvondouche** [comment]
> > Hi, would love to see support for Pollinate!!
> 
> seems like this is a MCreator. it's stated that these type of mods won't receive support

Ahh totally missed that, sorry about that!

**Comment by DarthCaesium** [comment]
There is a mod called [BetterNether](https://curseforge.com/minecraft/mc-mods/betternether), which I think would be a good mod to develop an add-on for. It adds 12 mushrooms, 2 cacti and 1 tree. However, it is currently only for Fabric, although a Forge port is in the works. For when the Forge port releases, could you please consider it? Once the Forge port is released, I will update my comment.

**Comment by rowanthemodder** [comment]
> 
> 
> [Mystical Biomes](https://www.curseforge.com/minecraft/mc-mods/mystical-biomes) - Sprout Forest for Mystical World, but oak trees that spawn are not dynamic.
> [More Minecarts and Rails](https://www.curseforge.com/minecraft/mc-mods/more-minecarts) - Glass Cactus that can only be planted in desert type biomes.

Mystical Biomes already has a dynamic trees add-on made by Nooby [Link](https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-mystical-biomes)

**Comment by WenXin20** [comment]
[Water Source](https://www.curseforge.com/minecraft/mc-mods/water-source) - A thirst mod that adds a Coconut Tree with coconut fruit to beaches.

**Comment by rowanthemodder** [comment]
[Horticulture](https://www.curseforge.com/minecraft/mc-mods/horticulture)
It's a part of the updated harvest festival modules and has 4 fruit bearing trees.

**Comment by juraj-hrivnak** [comment]
# [Charcoal Pit](https://www.curseforge.com/minecraft/mc-mods/charcoal-pit)
[**GitHub link**](https://github.com/EnderiumSmith/CharcoalPit2)

- It's a finely tuned mod that adds some features from TerraFirmaCraft.
It has **6 trees**.

**Comment by DarthCaesium** [comment]
[Advanced Rocketry](https://www.curseforge.com/minecraft/mc-mods/advanced-rocketry) - 3 trees

**Comment by Tyrius1** [comment]
Advent of Ascension. A mod that adds 21 dimensions with 29 different trees.

**Comment by Brun333rp** [comment]
[Twist](https://www.curseforge.com/minecraft/mc-mods/twist/screenshots)
Adds a Charred Forest biome and 1 type of tree

**Comment by Brun333rp** [comment]
[Twist](https://www.curseforge.com/minecraft/mc-mods/twist)
Adds 2 types of trees: Charred Tree (with no leaves) and Charred Spruce

**Comment by Brun333rp** [comment]
[World of Wonder](https://www.curseforge.com/minecraft/mc-mods/world-of-wonder) - adds 2 variants of giant Dandelions wich stems can used as wood

**Comment by Royal-Jelly** [comment]
I dove further into [infernal expansion](https://www.curseforge.com/minecraft/mc-mods/infernal-expansion) and have an interesting idea
<details>
 <summary>Reveal Screenshot</summary>

 ![2021-09-05_16 42 29](https://user-images.githubusercontent.com/14046681/132140920-9fe0d77c-fad9-4391-a20c-522a17baa48a.png) 

</details>
In the glowstone canyon biome brown mushrooms appear to grow from the ceiling.

Would it be feasible to rotate dynamic 'trees' to grow on the ceiling? If not that's fine.

**Comment by Harleyoc1** [comment]
> I dove further into [infernal expansion](https://www.curseforge.com/minecraft/mc-mods/infernal-expansion) and have an interesting idea
> 
> Reveal Screenshot
> In the glowstone canyon biome brown mushrooms appear to grow from the ceiling.
> Would it be feasible to rotate dynamic 'trees' to grow on the ceiling? If not that's fine.

Pretty sure the Natura add-on does exactly that for a nether tree in 1.12, so sure, when mushrooms are added to DT+ this could be implemented. 👍🏻

**Comment by Atlantispy** [comment]
Partly commenting so I get notified of updates to this, as I'd like to see Oh the Biomes and Pam's HarvestCraft 2 implemented.

**Comment by Sethyboi1** [comment]
[Primal Winter](https://www.curseforge.com/minecraft/mc-mods/primal-winter)  would be a good and easy mod to 
Add compatibility too as it simply replaces all the vanilla logs and trees to be have a snowier texture.

**Comment by XxDuckyUxX** [comment]
I have been waiting on Advent of Ascension to get a Dynamic Trees update  ever since I found out about both mods back in 2019. I surely do hope you make it.

**Comment by joe-vettek** [comment]
https://www.curseforge.com/minecraft/mc-mods/project-rankine

Rankie is also good.
![image](https://user-images.githubusercontent.com/82761814/149645780-e98948b5-4f72-4445-839b-87fbc5204990.png)
